src/views/Project/index.jsx

```diff
@@ -71,19 +132,72 @@ export default class Project extends Component {
   };
 
   componentDidMount() {
-    this.load();
+    this.loadMore();
   }
 
-  fetch = searchQuery => {
+  loadMore = () => {
+    if (this.props.bugzilla && this.props.bugzilla.loading) {
+      setTimeout(this.loadMore, 50);
+
+      return;
+    }
+
+    this.load();
+  };
```

Comment:
You can use a React lifecycle method to detect prop changes instead of using `setTimeout`. Remove `loadMore` and `componentDidMount` and give the following a try:

```jsx
componentDidUpdate(prevProps) {
  if (prevProps.bugzilla.loading && !this.props.bugzilla.loading) {
    this.load();
  }
}
```

Reply:
Ok! Added some logic to handle case where no bugzilla product exist (all github query)